club-website/src/components/header.tsx

```diff
@@ -1 +1,62 @@
-//test comment
\ No newline at end of file
+"use client"
+import * as React from "react"
+import Link from "next/link"
+import Image from "next/image";
+import {
+    NavigationMenu,
+    NavigationMenuContent,
+    NavigationMenuIndicator,
+    NavigationMenuItem,
+    navigationMenuTriggerStyle,
+    NavigationMenuLink,
+    NavigationMenuList,
+    NavigationMenuTrigger,
+    NavigationMenuViewport,
+  } from "@/components/ui/navigation-menu"
+
+export default function Header() {
+
+ 
+    return (
+        
+        <div className="flex justify-between p-6" id="logo_address">
+            <Link href="/about" legacyBehavior passHref>
+                <Image id="logo" src="/lightLogo.png" alt="" width={160} height={80}></Image> 
+            </Link>
```

Comment:
I don't think you need `legacyBehavior passHref` here, so you should get rid of it. Also I think the logo should link back to the root page so "/".

Reply:
done ! :D